runtime/jcl/common/jdk_internal_foreign_abi_UpcallStubs.cpp

```diff
@@ -65,11 +65,9 @@ Java_jdk_internal_foreign_abi_UpcallStubs_freeUpcallStub0(JNIEnv *env, jobject r
 
 	PORT_ACCESS_FROM_JAVAVM(vm);
 
-	omrthread_monitor_enter(vm->thunkHeapWrapperMutex);
+	omrthread_monitor_enter(vm->thunkHeapListMutex);
```

Comment:
The signature of this method is wrong; the second argument should have type `jclass` because it implements a `static` method.

Reply:
Agreed and fixed as suggested above.

Reply:
The declaration in `runtime/oti/jclprots.h` needs to be updated as well.

Reply:
Fixed.